### PR TITLE
Add Subprojects template.

### PR DESCRIPTION
**Pull request by jberkus**
As promised.

Dawn, April, Charles, please review!

### CODE REVIEW

**Comment by jberkus** [comment]
Hmmm, apparently I can't assign people.

@geekygirldawn @thisisnotapril

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:117]: This sentence seems awkward to me: Maybe:
These individuals will be chosen based on their community management and code of conduct
experience with diverse representation across the committee, including employer, gender,
race, background, and region of the world.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:150]: GitHub

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:6]: Once they adopt it, yes.  They haven't yet.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:13]: Yes ... again, I'm waiting for that template to get merged into Main.  It hasn't been.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:3]: federation?

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:39]: Aha!  Now, that's an important question because it is NOT the same person.  So apparently additional clairity is required!

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:55]: so the maintainers in the section above are not the maintainer representative.  different person.

clearly I need to do some additional labelling there.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:143]: The idea is for the *project* to vote by consensus, not the SC.

Comment (geekygirldawn) [reply GOVERNANCE-subprojects.md:143]: I'm not sure that I understand how a project can vote? Is it the members of the projects who can vote? Does each project get one vote on new projects? Or do you mean that the incoming project must have consensus from their members that they want to join?

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:17]: I used "federation".

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:6]: Cannot update per above, so resolving.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:13]: Cannot update per above, so resolving

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:3]: updated

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:96]: Done.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:117]: thanks, replaced

**Comment by jberkus** [comment]
@geekygirldawn all changes you recommended that are possible to incorporate have been added.  Thanks, please review for merge!

**Comment by geekygirldawn** [comment]
I can't seem to see your updates. I see that there is a new commit, but I don't see any changes when I look at the commit. I also get this error message: "This commit does not belong to any branch on this repository, and may belong to a fork outside of the repository."

**Comment by jberkus** [comment]
Dammit.  Git surgery required, apparently.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:47]: I would uncapitalize Of

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:21]: This looks like extra whitespace that we don't need?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:4]: On reading this again with your changes, I think some of my earlier confusion stems from how we are using project and subproject. I recommend consistently using "project" to refer to the collection (like Konveyor Project) and always say "subproject" when we refer to an individual project within the collection. I find myself getting confused about whether something refers to the group or subproject.

In this case: "... semi-independent subprojects"

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:17]: individual subprojects?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:29]: that subproject's Maintainer Committee, which governs that subproject.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:35]: subproject contributors?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:41]: Additionally, each subproject's Maintainer Committee ... 

I like replacing "the" with "each" in this case to make it clear that every subproject does this independently.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:56]: from each member subproject

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [reply GOVERNANCE-subprojects.md:73]: I think you missed this one in the last update?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:74]: Maybe "Resolving issues that individual subprojects are unable to resolve"?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:77]: and its subprojects.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:93]: I think this changed to "Community Representatives"?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:94]: of all subprojects

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:102]: sponsor might be ambiguous - should this be employer? And this last item doesn't make sense in the case of a single election office. I think maybe it should be "between two and three Election Officers" anyway, since it's good not to have a single point of failure during an election.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:124]: you have some extra punctuation here :)

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:139]: Adding New Subprojects?

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:154]: assist the new subproject in ...

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:157]: I would say "Experimental Subprojects" in both cases here

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:160]: but the subproject is not

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:161]: Experimental Subprojects

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:162]: full subproject status.

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:164]: Removing Subprojects

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:166]: In some cases, subprojects will

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:167]: removal of a subproject on

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:170]: Subprojects which still have ...

File: GOVERNANCE-subprojects.md
Comment (geekygirldawn) [inline GOVERNANCE-subprojects.md:172]: Subprojects which have ...

**Review by geekygirldawn — CHANGES_REQUESTED** [review]
This review got really long, but I think I figured out that inconsistent use of "project" and "subproject" language was the root of some of my confusion because it because hard to tell when the document is referring to the project as a whole vs. referring to the subprojects. I've made suggestions that try to make this clear.

Overall, I think this is looking really good!

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:4]: This has been updated across the file, including replacing 40 instances of "project" with "subproject".

**Comment by jberkus** [comment]
@geekygirldawn  I think I got every instance of project/subproject, please check.  Thanks!

**Comment by geekygirldawn** [comment]
@jberkus I think you got all of the project / subproject changes, but it looks like there are still a few outstanding suggestions outside of that change?

**Comment by jberkus** [comment]
The only other one I see is on the extra line.  Anything else?

File: GOVERNANCE-subprojects.md
Comment (saad-ali) [inline GOVERNANCE-subprojects.md:1]: We should add a disclaimer saying something like `This is a suggested template and not required by the CNCF` to avoid any confusion (don't want us to be accused of "forcing a governance model").

**Review by saad-ali — COMMENTED** [review]
/lgtm

File: GOVERNANCE-subprojects.md
Comment (saad-ali) [reply GOVERNANCE-subprojects.md:1]: Might make more sense to stick such a disclaimer on https://github.com/cncf/project-template/blob/main/README.md rather then each template.

Comment (jberkus) [reply GOVERNANCE-subprojects.md:1]: Yeah, probably.  I mean, the fact that there are multiple templates available would seem to indicate that, but one can't be too careful.

File: GOVERNANCE-subprojects.md
Comment (joshuagl) [inline GOVERNANCE-subprojects.md:84]:
```suggestion
that is open to all project Members.  The Committee may hold additional, closed meetings
```
It took me a few reads of this section to infer that we're talking about all project Members (subproject maintainers) here.

File: GOVERNANCE-subprojects.md
Comment (joshuagl) [inline GOVERNANCE-subprojects.md:132]: Should there be a [TODO: ] entry here to include the confidential reporting mechanism?

**Review by joshuagl — COMMENTED** [review]
I hope you don't mind the drive-by review. There were a few pieces of this that weren't clear to me on first read through, so I've suggested some minor clarifications in addition to those already suggested by @geekygirldawn.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:84]: not just maintainers, but all contributors.  Changing wording.

File: GOVERNANCE-subprojects.md
Comment (jberkus) [reply GOVERNANCE-subprojects.md:128]: I kinda wanna leave this to the CoC document, and not replicate it here.  This document is concerned with the selection of the CoC rather than their mechanics.

**Comment by jberkus** [comment]
OK, I **think** all comments are resolved.

**Comment by geekygirldawn** [comment]
Thanks so much for doing all of this work! 

/lgtm

**Comment by jberkus** [comment]
Approved at March 16 Gov-WG meeting.  @carolynvs please merge.

**Comment by carolynvs** [comment]
Yay!

**Pull request merged**
